setup.py

```diff
@@ -44,32 +41,39 @@ def package_data(pkg, root_list):
 
 BLOCKS = [
     'mentoring = mentoring:MentoringBlock',
-    'mentoring-dataexport = mentoring:MentoringDataExportBlock',
-]
 
-BLOCKS_CHILDREN = [
     'mentoring-table = mentoring:MentoringTableBlock',
-    'column = mentoring:MentoringTableColumnBlock',
-    'header = mentoring:MentoringTableColumnHeaderBlock',
+    'mentoring-column = mentoring:MentoringTableColumn',
     'answer = mentoring:AnswerBlock',
-    'quizz = mentoring:QuizzBlock',
-    'message = mentoring:MentoringMessageBlock',
-    'tip = mentoring:QuizzTipBlock',
-    'choice = mentoring:QuizzChoiceBlock',
-    'html = mentoring:HTMLBlock',
+    'answer-recap = mentoring:AnswerRecapBlock',
+    'mcq = mentoring:MCQBlock',
+    'rating = mentoring:RatingBlock',
+    'mrq = mentoring:MRQBlock',
+    'mentoring-message = mentoring:MentoringMessageBlock',
+    'tip = mentoring:TipBlock',
```

Comment:
Well there is [xblock-adventure](https://github.com/edx-solutions/xblock-adventure) which re-uses the MCQ element from this, but I generally agree with you. @antoviaque what do you think?